nohup: ignoring input
2025-08-15 14:39:58,847 - database - INFO - Database initialized successfully
2025-08-15 14:39:58,912 - root - INFO - Fake Email Bot başladıldı...
2025-08-15 14:39:59,183 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getMe "HTTP/1.1 200 OK"
2025-08-15 14:39:59,268 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/deleteWebhook "HTTP/1.1 200 OK"
2025-08-15 14:39:59,269 - telegram.ext.Application - INFO - Application started
2025-08-15 14:40:09,531 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:40:19,616 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:40:29,704 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:40:39,790 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:40:49,880 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:40:59,970 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:41:00,055 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:41:00,063 - root - ERROR - Update Update(message=Message(channel_chat_created=False, chat=Chat(first_name='Қəɳαɳ', id=1143980741, type=<ChatType.PRIVATE>, username='kenanbabayevofficial'), date=datetime.datetime(2025, 8, 15, 14, 40, 59, tzinfo=datetime.timezone.utc), delete_chat_photo=False, entities=(MessageEntity(length=6, offset=0, type=<MessageEntityType.BOT_COMMAND>),), from_user=User(first_name='Қəɳαɳ', id=1143980741, is_bot=False, language_code='tr', username='kenanbabayevofficial'), group_chat_created=False, message_id=760, supergroup_chat_created=False, text='/start'), update_id=459449801) caused error name 'SUPPORTED_LANGUAGES' is not defined
2025-08-15 14:41:10,150 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
